content/blog/aosus-v3/index.ar.md

```
@@ -0,0 +1,63 @@
+thumbnail: "aosus-v3.jpg"
+ogImage: "aosus-v3.jpg"
+excerpt: "الإصدار الثالث من مجتمع أسس يأتي بهوية جديدة، موقع جديد، وتجارب أذكى للمجتمع ومنصاته."
+slug: "aosus-v3"
+lang: "ar"
+---
+
+السلام عليكم ورحمة الله وبركاته، رحبوا معنا بالإصدار الثالث من مجتمع أسس.
+
+هوية جديدة، وبداية جديدة في عصر الذكاء الاصطناعي.
+
+## هوية مجتمع أسس الجديدة
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)